### DISCUSSION TITLE
Improvement to search bar using serve

### DISCUSSION THREAD

**Discussion by rleaver152** [policratus/pupyl#128] [Ideas]
Many searches are likely to be local filesystem

When you drag a local file to be searched into the "Place an image url here for search" field,   it would be very help to automatically have it prefixed with file://     so that it is recognised as a local file.         Maybe this can be ignored if dropping from a web browser for an external site.    

At the moment it has to be manually inserted which whilst not tedious,  is repetitive for multiple searches.

**Comment by policratus** [comment]
Great idea @rleaver152, this is an enhancement that some people are waiting for.

Would you have some idea on how to implement that? If yes, could you help us on that? We can add you as a `pupyl` commiter and you can open a pull request implementing this.

What you think?

**Comment by rleaver152** [comment]
Happy to try - I’m only really bash/Python tbh

R

**Reply by policratus** [reply]
That's ok @rleaver152, I'll try to implement it and publish also on version `0.13.2`.
